app/models/ExportsDeclaration.scala

```diff
@@ -81,8 +81,8 @@ case class ExportsDeclaration(
   def isEidr: Boolean = parties.isEntryIntoDeclarantsRecords.exists(_.answer == YesNoAnswers.yes)
   def isNotEidr: Boolean = !isEidr
 
-  def isEntryIntoDeclarantsRecords: Boolean = parties.isEntryIntoDeclarantsRecords.exists(_.answer == YesNoAnswers.yes)
-  def isNotEntryIntoDeclarantsRecords: Boolean = !isEntryIntoDeclarantsRecords
+  def isEntryIntoDeclarantsRecords: Boolean = isEidr
+  def isNotEntryIntoDeclarantsRecords: Boolean = !isEidr
```

Comment:
These methods seem redundant, can we not replace them and then delete them?

Reply:
They were already there. I left them because maybe someone was preferring the extended name